### ISSUE TITLE
[Bug][a11y]: Card action menu button  should have label

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3388]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Card

### Actual behavior

In the Card documentation and storybook examples that include an action menu, the `sp-action-menu` should have a `label` attribute, or if not, a default label should be provided.

See: https://github.com/adobe/spectrum-web-components/blob/8eb28c866c960b2cf27930cc1eab8f96e2c057d9/packages/card/README.md?plain=1#L110-L118 or https://github.com/adobe/spectrum-web-components/blob/8eb28c866c960b2cf27930cc1eab8f96e2c057d9/packages/card/README.md?plain=1#L184-L192

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
